src/core/main.lua

```diff
@@ -11,7 +11,6 @@ local ffi = require("ffi")
 local zone = require("jit.zone")
 local lib = require("core.lib")
 local shm = require("core.shm")
-local app = require("core.app")
```

Comment:
I want to draw attention to this change specifically.

I was noticing a bug in the code where upon running a program such as "top" with no arguments and no other processes running it'd still connect to a process and display the counter information. It turned out because we were including app in main.lua and app at the top level sets up a counter directory for the engine the supervisor process actually had their own PID directory which was being created a not cleaned up. This also caused an issue that the process top was connecting to was it's supervisor process which was a little nonsensical.

I looked through the code and verified nothing seems to be using the import, removing it and running all the tests worked fine and also trying to run a few snabb programs seems to work too. Not sure if this was left in there from a time when it was used.

I just want to check there is no other magic that it's doing and it's safe to remove.